MapleServer2/Types/TriggerObject.cs

```diff
@@ -47,4 +49,48 @@ public TriggerActor(int id, bool isVisible, string stateName) : base(id)
             StateName = stateName;
         }
     }
+
+    public class TriggerLadder : TriggerObject
+    {
+        public bool IsVisible;
```

Comment:
Don't need `IsVisible` if it's already in the parent object, same on line 68 and 81

Reply:
The parent object doesn't have `IsVisible`

Reply:
I think he meant to add `IsVisible` to **TriggerObject** ?

Reply:
Not all TriggerObjects use `IsVisible`

Reply:
My comment is gone, lol.. but sounds good!

Reply:
Oh, nevermind